lessons/Projects/project-function/langs/en-us/pages/code-your-function-2.adoc

```diff
@@ -0,0 +1,53 @@
+= Coding Your Function (2)
+
+Open @starter-file{editor} on your computer. Sign in and create a new file. Give it an appropriate name.
+
+== Add Examples and Define Your Function
+
+@n Translate your Circle of Evaluation into code, and type it into the Interactions Area, . Since you haven't defined your function yet, Pyret won't recognize `image` as your variable. You'll need to use a specific shape function (and its arguments!) instead. Write the code for the shape function and its arguments that you will use to test your code.
```

Comment:
We discussed this -- I think this page should be redone once added predefined shapes to the starter file.